browser/src/Services/SyntaxHighlighting/Definitions.ts

```diff
@@ -1,11 +1,11 @@
 import * as types from "vscode-languageserver-types"
 
 export interface IHighlight {
-    foregroundColor: string
-    backgroundColor: string
-
+    foreground: string
+    background: string
     bold: boolean
     italic: boolean
+    fallback?: string
```

Comment:
With the latest change, is this still needed?

Reply:
ah thats definitely not needed its was a point I missed, no good find and delete in project yet maybe another editor feature 😆